provider/cpu_utils.py

```diff
@@ -11,17 +12,25 @@ class VMStressBinding(VMStress):
     def __init__(self, vm, params, stress_args=""):
         super(VMStressBinding, self).__init__(vm, "stress", params,
                                               stress_args=stress_args)
-        self.install()  # pylint: disable=E0203
-        self.install = lambda: None
+        self.install()
 
     def load_stress_tool(self, cpu_id):
         """
         Load the stress tool and bind it to the specified CPU
 
         :param cpu_id: CPU id you want to bind
         """
-        self.stress_cmds = "taskset -c %s stress" % cpu_id  # pylint: disable=W0201
-        super(VMStressBinding, self).load_stress_tool()
+        cmd = "setsid taskset -c {} {} {} > /dev/null".format(cpu_id,
```

Comment:
Why not update load_stress_tool() directly?

Reply:
Since only this case currently has high cpu stress, and may encounter this problem.
VMStress on vt is invoked by many test cases, so if other users have not reported this issue temporarily, I will give priority to fix it only here. (I don't want this modification to affect other cases.)
BTW, the parent `load_stress_tool()` will call `self.install()` every time, and we need to bind cpu here, So I decide to rewrote it instead of inheriting from the parent method.

Reply:
Make sense. ACK.